src/glass/src/app/core/modals/nfs/nfs-modal.component.html

```diff
@@ -77,7 +77,7 @@ <h1 mat-dialog-title>NFS</h1>
                     fxFlex>
         </mat-slider>
         <mat-hint>
-          <span>{{formGroup.get('replicas')?.value | redundancyLevel:'long'}}</span>
+          <span>{{ formGroup.get('replicas')?.value | redundancyLevel:'long' }}</span>
```

Comment:
We do have the same line on the `CephfsModalComponent`: https://github.com/aquarist-labs/aquarium/blob/main/src/glass/src/app/core/modals/cephfs/cephfs-modal.component.html#L80

Reply:
Doesn't matter. We do not have formatting rules related to this. It's my personal preference to format such strings this way. I changed that only because i came across this code while doing the implementation for this PR. Angular itself seems not to have strict rules, they use both `{{foo}}`  and `{{ foo }}`, see [this example](https://angular.io/guide/interpolation#preventing-name-collisions).

Reply:
I fully agree with you. I'm also in favor of the latter.